helper/chroot/chroot.go

```diff
@@ -182,10 +161,6 @@ func (fs *ChrootHelper) Lstat(filename string) (os.FileInfo, error) {
 }
 
 func (fs *ChrootHelper) Symlink(target, link string) error {
-	if !fs.symlinkSupport {
-		return billy.ErrNotSupported
-	}
-
 	target = filepath.FromSlash(target)
```

Comment:
By the way, I've just realized about this: does this mean that OS-specific rules on paths apply to *every* filesystem wrapped in ChrootHelper? What if I'm on Windows but using a GCE backend, which uses Linux-style paths?